src/components/AndelaCentres/CentreModal.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Form } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import InputFluid from '../common/TextInputComponent';
+import ArtButton from '../common/ButtonComponent';
+import DropdownComponent from '../common/DropdownComponent';
+import StatusComponent from '../common/StatusComponent';
+import { generateModalDropdownOptions } from '../../_utils/generateDropdownOptions';
+
+const removeDuplicates = (locations) => {
```

Comment:
I believe you don't need this as there is a `/countries` endpoint. Ask Thuku about it